12_upgrade/00_upgrade.txt

```diff
@@ -10,16 +10,16 @@ automatically by packaging scripts. Therefore, you shouldn't have to do any
 upgrade procedures manually, but you will note that several data migrations
 occur during the upgrade process.
 
-==== Upgrade from Rudder 2.11, 3.0 or 3.1
+==== Upgrade from Rudder 3.1 or 3.2
 
-Migration from 2.11, 3.0 or 3.1 are supported, so you can upgrade directly to 3.2.
+Migration from 3.1 or 3.2 are supported, so you can upgrade directly to 4.0.
 
-If you migrate from 2.11, 3.0 or 3.1 older than 3.1.11, you have to upgrade
-you server before your agents (as 3.2 agents are not compatible with older servers).
+If you migrate from a 3.1 release older than 3.1.11, you have to upgrade
+you server before your agents (as 4.0 agents are not compatible with old servers).
```

Comment:
Yes, because https://github.com/Normation/rudder-techniques/pull/830 was already done in 3.2.0.